### DISCUSSION TITLE
Questions about customizing debugger extension.

### DISCUSSION THREAD

**Discussion by mushahidkhan835** [microsoft/vscode-discussions#1018] [Extension Development QnA] [answered]
Hi all,
I am a beginner in developing extensions. I have some questions I hope someone can answer :) 
If i take a debugger, https://code.visualstudio.com/api/extension-guides/debugger-extension, can i customize the existing settings of the debugger to display things such as images in my debugger. Also, can i get access to the stack trace with line numbers and current line on a break point?

thank you so much

**Comment by isidorn** [comment, marked as answer]
Yes, this should be possible. That guide should be helpful. Let us know if you hit any issues. Thank you

**Comment by mushahidkhan835** [comment]
Hi I am following the tutorial but i cant seem to understand it well. I am trying to build a debugger where anytime it sees a comment #end here the code, I should see show a picture. Is there another tutorial for building a debugger extension or some video tutorial or some other project I can reference?
